### ISSUE TITLE
Feature Request: ability to merge 2 or more groups on a easy fashion?

### ISSUE THREAD

**Issue by aniel300** [SenexCrenshaw/StreamMaster#55]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
ability to merge 2 or more groups on a easy fashion? i am yet to use this option but i can see myself in the future doing and needing this. i also think it would be a nice addition to SM

**Describe alternatives you've considered**
maybe manually move channels back and fort from one group to the other as needed. but this is not efficient nor i think it possible at the moment with sm. 

**Additional context**
n/a

**Comment by SenexCrenshaw** [comment]
Why would you need this? Can you just use multi-select check boxes?

**Comment by aniel300** [comment]
yes but what else can i do after that? does sm have a merged function already?

**Comment by aniel300** [comment]
this is part of my "multi provider" support initiative. i need some interest from you on this to make sure we are in the same page

**Comment by SenexCrenshaw** [comment]
i am not sure what you are looking for. A group of groups?

**Comment by aniel300** [comment]
well the new request include something extra but in a nutshell, i would like to be able to lets say group a bro provider a with group b from provider b, etc. as well be able to rename/name that new group on the fly if need (lest call that new group c for the sake of this example) to but then i would like sm to automatically sync new content/channels added to group a and b to c and prevent group a and b from being regenerated on playlist refresh something similar to this from iptveditor: 
![image](https://github.com/SenexCrenshaw/StreamMaster/assets/7571729/9c6eecea-7e52-4a8b-8d09-658058301736)

thank you.
